jupyter_server/services/sessions/sessionmanager.py

```diff
@@ -268,15 +286,52 @@ async def create_session(
         self._pending_sessions.remove(record)
         return result
 
-    def get_kernel_env(self, path):
-        """Return the environment variables that need to be set in the kernel"""
+    def get_kernel_env(
+        self, path: Optional[str], name: Optional["ModelName"] = None
```

Comment:
`ModelName` should not be quoted here.
```suggestion
        self, path: Optional[str], name: Optional[ModelName] = None
```